react/src/index.tsx

```diff
@@ -20,6 +20,10 @@ import CallCount from "./components/CallCount";
 import APIForm from "./components/APIForm";
 import Settings from "./components/Settings";
 import GroupCallCount from "./components/GroupCallCount";
+import Bugsnag from "@bugsnag/js";
+
+Bugsnag.start("67e3931dbe1bbf48991ce7d682ceb676");
```

Comment:
I think the tradeoff (of possibly missing bugs from external scripts) is worth the potential benefit of a faster load time, but ultimately that's up to you

Reply:
There's not a whole lot of useful signal in bugsnag right now but I worry that there could be and we'd miss it? I dunno if there's a lighter weight way to do it that is not the whole bugsnag suite.